README.md

```diff
@@ -16,9 +16,9 @@
 
 ## Why DuraCloud?
 
-In the digital era, ensuring that critically important documents remain safe and available is a continual challenge. Physical computing hardware that is used to create and store documents can fail or become obsolete very quickly, providing a clear need for decisive action to ensure that these documents remain available.
+In the digital era, ensuring that critically important documents remain safe and available is a continual challenge. Physical computing hardware that is used to create and store documents can fail or become obsolete very quickly.
 
-There are many options for file storage and backup, with a growing trend toward the use of service providers offering off-site storage and backup solutions. The convenience and elasticity of these solutions is enticing, but several concerns often remain:
+There are many options for file storage and backup, with a growing trend toward the use of service providers offering off-site storage and backup solutions. However, several concerns often remain:
```

Comment:
It seems that "concerns often remain" makes less sense without the introductory clause here. From what are the concerns remaining? I'm fine with removing the text, but the remaining sentence may need to be reworked.

Reply:
You're right. I used a machete when a scalpel would do! Would removing "The convenience and elasticity of" work? None of these cuts are necessary. I was just seeing if we could make it a little shorter.